.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules/*
 /client/libs/*
 /sessions
+/repos/*
```

Comment:
We have a defaulf repo commited that is the demo one. I think It does not make sense putting repos in .gitignore. Unless we implemented this feature -> https://github.com/solettaproject/soletta-dev-app/issues/82 then we could add repos into .gitignore file.